File: src/main/java/tk/avabin/tdg/beans/entities/RPGClassAndLevel.java
package tk.avabin.tdg.beans.entities;

import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.context.annotation.Scope;
import org.springframework.stereotype.Component;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.Table;

/**
 * Created by Avabin on 10.04.2017.
 */
@Component
@Scope("prototype")
@Data
@NoArgsConstructor
@Entity
@Table(name = "rpg_class")
public class RPGClassAndLevel {
    private String rpgClass;
    private Integer level;

    @Id
    @GeneratedValue
    private int id;
}

File: src/main/kotlin/tk/avabin/tdg/beans/entities/RPGClassAndLevel.kt
package tk.avabin.tdg.beans.entities

import org.springframework.context.annotation.Scope
import org.springframework.stereotype.Component
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id
import javax.persistence.Table

/**
 * Created by Avabin on 10.04.2017.
 */
@Component
@Scope("prototype")
@Entity
@Table(name = "rpg_class")
data class RPGClassAndLevel(
        var rpgClass: String = "",
        var level: Int = 0,

        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        var id: Int = 0
)